src/pykeen/models/base.py

```diff
@@ -563,6 +563,27 @@ def predict_scores_all_relations(
             scores = torch.sigmoid(scores)
         return scores
 
+    def _score_with_inverse_relations(
+        self,
+        hrt_batch: torch.LongTensor,
+    ) -> torch.FloatTensor:
+        """Score triples based on inverse triples, i.e., compute f(h,r,t) based on f(t,r_inv,h)."""
+        if not self.triples_factory.create_inverse_triples:
+            raise Exception("Model is not configured to predict with inverse relations. "
```

Comment:
ValueError

Reply:
Fiixed in 8fbbc660958bc6c3dc76fa56f84a16960fc14be7.